examples/kitchen-sink/apps/api/tsconfig.json

```diff
@@ -2,8 +2,8 @@
   "extends": "@repo/typescript-config/base.json",
   "compilerOptions": {
     "lib": ["ES2015"],
-    "module": "CommonJS",
-    "outDir": "./dist",
+    // "module": "CommonJS",
```

Comment:
Was this intentional?

Reply:
yes. module:"CommonJs" was causing an error as the moduleResolution from base.json is set to Bundler. I commented line 5 so that it could use module:"ESNext" from base.json. If there is a better fix for that please let me know.

Reply:
Could you delete it instead of commenting it out?

Reply:
yes of course

Reply:
@anthonyshew done